### DISCUSSION TITLE
Is there any way to make changes to the BMS using this software?

### DISCUSSION THREAD

**Discussion by gyrex** [txubelaxu/esphome-jk-bms#27] [General]
Firstly, thanks for this wonderful project. I had lots of issues with the BLE project.

I can see in the docs that the switches are read-only, is there any way to make them actually write changes to the BMS?

**Comment by txubelaxu** [comment]
The actual version is read & write. There is a "broadcast mode". If it is "on" same change is sent to every BMS.

Thank you very much!!
